### PR TITLE
Sizing

### PR DESCRIPTION
**Pull request by hdua1**
Sizing and distance need to be handled together. I'll first merge this working version (containing some part of sizing code) with dev and branch off of dev to work on both together.

### CODE REVIEW

**Comment by hdua1** [comment]
Perfect. Would work for distancing. I'll merge with dev.

File: src/js/index.js [hunk 1]
```diff
@@ -51,6 +53,7 @@ scene.add( axesHelper );
 let geometry = new THREE.SphereGeometry( 3, 32, 32 );
 let material = new THREE.MeshPhongMaterial( {color: 0xffff00, wireframe: true} );
 let sphere = new THREE.Mesh( geometry, material );
+sphere.scale.set(50,50,50);
```
Comment (trevorforrey) [inline src/js/index.js]: Would you be able to add the size (50) as a constant at the top of the index.js?

File: src/js/models/earth.js [hunk 2]
```diff
@@ -20,8 +20,9 @@ export default class Earth {
                 map:    earthTexture,
             });
             let earthMesh = new THREE.Mesh(earthGeo, earthMaterial);
-            earthMesh.rotation.x += 0.5;
-            earthMesh.position.x += 5;
+            earthMesh.scale.set(5,5,5);
```
Comment (trevorforrey) [inline src/js/models/earth.js]: Doing the same for earth size could help too. Have it be a constant in index.js and then pass it into the constructor

File: src/js/models/moon.js [hunk 3]
```diff
@@ -15,12 +15,13 @@ export default class Moon {
...
     load() {
         return loadTexture(totallyMoonTexture, new THREE.TextureLoader()).then((moonTexture) => {
-            let geometry = new THREE.SphereGeometry(this.size, 32, 32);
+            let geometry = new THREE.SphereGeometry(this.size, 10, 10);
             let material = new THREE.MeshBasicMaterial({
                 map:    moonTexture,
             });
             let moon = new THREE.Mesh(geometry, material);
-            moon.position.x += 6;
+            moon.scale.set(1.25,1.25,1.25);
```
Comment (trevorforrey) [inline src/js/models/moon.js]: Same thoughts here as with the earth object.

**Review by trevorforrey — CHANGES_REQUESTED** [review]
Looks good, I just have a few suggestions before merging it into dev.

**Review by trevorforrey — APPROVED** [review]

**Pull request merged**